src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -23,9 +23,17 @@ const STRIP_VIEW_SELECTORS = {
     ARRIVAL_AIRPORT_ID: '.js-stripView-arrivalAirportId',
     ALTERNATE_AIRPORT_ID: '.js-stripView-alternateAirportId',
     FLIGHT_PLAN: '.js-stripView-flightPlan',
-    REMARKS: '.js-stripView-remarks'
+    REMARKS: '.js-stripView-remarks',
+    RUNWAY: '.js-stripView-runway'
 };
 
+/**
+ * @property STRIP_VIEW_PREPLANNING_CLASS
+ * @type {string}
+ * @final
+ */
+const STRIP_VIEW_PREPLANNING_CLASS = "stripView-preplanning";
```

Comment:
this should live in either `STRIP_VIEW_SELECTORS` above or in the main `SELECTORS` object.  Also, no double quotes, only single quote.

Reply:
Don't we have a constants file for classes? That's where I would think it would belong.

Reply:
we have `CLASSNAMES` in `selectors.js`

Reply:
Okay, exactly. So it would belong there then, no?

Reply:
moved there